src/tbp/monty/frameworks/run_env.py

```diff
@@ -31,6 +31,13 @@ def setup_env(monty_logs_dir_default: str = "~/tbp/results/monty/"):
         os.environ["MONTY_MODELS"] = monty_models_dir
         print(f"MONTY_MODELS not set. Using default directory: {monty_models_dir}")
 
+    monty_data_dir = os.getenv("MONTY_DATA")
+
+    if monty_data_dir is None:
+        monty_data_dir = os.path.expanduser("~/tbp/data/")
```

Comment:
I noticed that in this instance you call `os.path.expanduser` once, whereas for the other directories, we don't, requiring us to call it every time e.g. `monty_logs_dir` is used. 

I think the way you've done it here looks a bit cleaner, something we can think about.